src/sequences/SequenceRandom.ts

```diff
@@ -0,0 +1,77 @@
+import { ValidationStatus } from '@/shared/ValidationStatus';
+import { SequenceParamsSchema, SequenceExportModule,
+         SequenceExportKind } from './SequenceInterface';
+import { SequenceCached } from './SequenceCached';
+
+/**
+ *
+ * @class SequenceClassRandom
+ * Creates a sequence of random integers in a specified range.
+ *
+ */
+class SequenceRandom extends SequenceCached {
+	name = "Random Integers in Range";
+	description = "A sequence of integers chosen indepenently uniformly from n to m inclusive.";
+	params: SequenceParamsSchema[] = [new SequenceParamsSchema(
+		'min',
+		'number',
+		'Minimum value attainable',
+		false,
+		0 
+	),
+	new SequenceParamsSchema(
+		'max',
+		'number',
+		'Maximum value attainable',
+		false,
+		9
+	)];
+
+	private minimum = 0;
+	private maximum = 0;
+
+	/**
+     	*Creates an instance of SequenceRandom
+     	* @param {*} ID the ID of the sequence
+	*/
+	constructor (ID: number) {
+		super(ID);
+	}
+
+	validate(){
+		this.settings['name'] = 'Constant';
+		const superStatus = super.validate();
+		if (!superStatus.isValid) {
+			return superStatus;
+		}
+
+		if (this.settings['min'] == undefined || this.settings['max'] == undefined ) {
+			this.isValid = false;
+			return new ValidationStatus(false, ["The min or max parameter is missing."]);
+		}
+
+		if ( !Number.isInteger( Number(this.settings.min) ) ) return new ValidationStatus(false, ["The min value must be an integer."]);
+		if ( !Number.isInteger( Number(this.settings.max) ) ) return new ValidationStatus(false, ["The max value must be an integer."]);
+
```

Comment:
It's an unfortunate design flaw in the implementation of SequenceClassDefault that if either of the above cases are hit, `this.isValid` will be true: it's set to true in SequenceClassDefault.validate() and not reset to false in either of these cases, unlike in the cases where 'min' or 'max' is undefined. I'll create an issue for this design flaw, but in the meantime, the expedient thing to do for this PR would be to set this.isValid to false immediately after checking the return from super.validate() like at line 47. Then you can go through the bad conditions, with the isValid flag safely false if any of them are hit (i.e. you can then remove line 49) , and then you already set it back to true below once you've gotten through all the bad conditions, so it will be OK if there are no bad conditions. Hope I am making sense.